wfl/Database/MongoDBManager.cs

```diff
@@ -19,69 +20,60 @@ public MongoDBManager()
             db = client.GetDatabase("WFL");
         }
 
-        public async Task<Restaurant> InsertRestaurantAsync(RestaurantDto r)
+        private string GenerateCollectionName<T>()
```

Comment:
As a follow up refactoring here, I think it would be nice to remove this logic from the DBManager (or repository) all together.  This value never changes and we can set it at the time that we create the db manager.  It would be nice to compute it once, and even nicer if it was passed into this class making it a simpler implementation.